extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelExtractor.java

```diff
@@ -137,19 +142,32 @@ public String getFeedUrl() throws ParsingException {
 
     @Override
     public long getSubscriberCount() throws ParsingException {
-
+        long subCount = -1;
         final Element el = doc.select("span[class*=\"yt-subscription-button-subscriber-count\"]").first();
         if (el != null) {
+            // If the element is null, the channel have the subscriber count disabled
             String elTitle = el.attr("title");
             try {
-                return Utils.mixedNumberWordToLong(elTitle);
+                subCount = mixedNumberWordToLong(elTitle);
             } catch (NumberFormatException e) {
                 throw new ParsingException("Could not get subscriber count", e);
             }
-        } else {
-            // If the element is null, the channel have the subscriber count disabled
-            return -1;
+
+            if (!getExtractorLocalization().getLanguageCode().equals("en") && subCount < 1000 && subCount != -1) {
```

Comment:
> Ah yes, I remember. In this case, the best solution would be to create a list of the abbreviations for all supported languages and then convert the numbers correctly. Making a new request for a single value seems not the correct approach to me as we cause a bunch of traffic.

I move the discussion here @TobiGr 
How should we do?
I thought that:

1. Do a HashMap with
+ Key: abreviation in the language, with the space (or not if the language stick it)
+ value: English equivalent
2. Replace key by its value in the string
3. use mixedNumberWordToLong function.

Do you think of another better way?

Reply:
> 1. Do a HashMap with
> - Key: abreviation in the language, with the space (or not if the language stick it)
> - value: English equivalent
>
>2. Replace key by its value in the string

I'd suggest to do this inside a second overloading `mixedNumberWordToLong(String numberWord, Locale locale)` method wich then calls the `mixedNumberWordToLong(String numberWord)` function.

Reply:
Done (wow that was hard and long)